SubjuGator/perception/subjugator_perception/nodes/vision_stack_test.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import rospy
+from image_geometry import PinholeCameraModel
+from mil_ros_tools import (
+    Image_Subscriber,
+)
+from vision_stack import ResizeLayer, UnderWaterImageEnhancementLayer, VisionStack
+
+__author__ = "Daniel Parra"
+
+
+class ObjectDetectionTest:
+    def __init__(self):
+        camera = rospy.get_param("~image_topic", "/camera/front/right/image_rect_color")
+        SIZE = (960, 608)
+        self.vs = VisionStack(
+            layers=[
+                ResizeLayer((0, 0), 960, 608),
```

Comment:
This should be
```python
ResizeLayer((0, 0), SIZE[0], SIZE[1])
```
right?

Reply:
The first argument is the initial size, and the next parameters are the new width and new heights. However, the initial size of the image is not being used in any of the layers...

Reply:
Ill change that rn